docs/pages/access-controls/access-request-plugins/ssh-approval-mattermost.mdx

```diff
@@ -174,12 +201,35 @@ files and leave `identity` commented out.
 `identity` field with the path to the identity file you exported earlier and
 comment out the other fields.
 
+</TabItem>
+<TabItem label="Teleport Enterprise " scope={["oss", "enterprise"]} options="Helm Chart">
+
+**`address`**: Include the hostname and HTTPS port of your Teleport Cloud tenant
+(e.g., `mytenant.teleport.sh:443`).
+
+**`identitySecretName`**: Fill in the `identitySecretName` field with the name
+of the Kubernetes secret you created earlier.
+If you exported an identity file earlier, fill in the `identity` field with the
```

Comment:
Just double-checking that this sentence is accurate for the Helm chart:

> If you exported an identity file earlier, fill in the identity field with the path to the file and comment out the other fields.

Reply:
Oh, forgot to delete that sentence